<launch>
  
  <node pkg="evaluation_ground_truth_annotator" type="annotate_symmetric_plane_node"
	 name="annotate_symmetric_plane" output="screen">
    <remap from="input_cloud" to="/camera/depth_registered/points" />
    <remap from="input_point" to="/camera/rgb/image_rect_color/screenpoint" />

    <remap from="input_gt_cloud" to="/selected_pointcloud" />
</node>


</launch>
